### PR TITLE
changes reverted

### PR DESCRIPTION
**Pull request by imAffanAli**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "license": "ISC",
+  "devDependencies": {
+    "@types/express": "^4.17.21",
+    "@types/node": "^20.14.9",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.5.2"
+  },
+  "dependencies": {
+    "express": "^4.19.2"
+  }
```
Comment (waqasCodeNinja) [inline package.json:23]: Please Specify Engines here also.

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import express, { Request, Response } from 'express';
+
+const app = express();
```
Comment (waqasCodeNinja) [inline src/index.ts:3]: Shouldn't we apply type here ?

Comment (imAffanAli) [reply src/index.ts]: added types for app and port

File: package.json (hunk 1 above)
Comment (imAffanAli) [reply package.json:26]: i set the engine req to >=18.0.0

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import express, { Request, Response ,Application} from 'express';
+
+const app: Application = express();
```
Comment (waqasCodeNinja) [inline src/index.ts:3]: shouldn't we use "Express" Type ?

Comment (imAffanAli) [reply src/index.ts:3]: Application and Express can be used interchangeably but I used Application as it makes it clear that it is a core express app both

**Pull request merged**